end_to_end/test_llama2_7b.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# This file is both an integration test that runs once a day on a v4-8 and documentation for how to get started with Llama2-7b
+
+# The flow of this file is as follows:
+# 1. Download the checkpoint from Meta (https://llama.meta.com/llama-downloads/) in your local directory. Convert this PyTorch checkpoint into Orbax checkpoint format for use in MaxText. 
+# 2. Run decoding, finetuning of Llama2-7b with this converted checkpoint. Also, run pretraining of Llama2-7b.
+# 3. Run decoding from the finetuned weights
+# 4. Convert the scanned checkpoint from step #1 into unscanned checkpoint format and run more efficient decoding. 
+
+
+set -ex
+idx=$(date +%Y-%m-%d-%H-%M)-${RANDOM}
```

Comment:
nit: remove -${RANDOM}, see Jon's PR: https://github.com/google/maxtext/pull/545/files

Reply:
done - thank you!